HW04/ivan_klimov/spec/person_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+require_relative '../lib/person'
+
+RSpec.describe Person do
```

Comment:
There is no need to test this class

Reply:
I deleted the test files for person and homework